src/__init__.py

```diff
@@ -1,3 +1,18 @@
 class Interpreter(object):
     def parse(self, text):
         raise NotImplementedError()
+
+
+config_keys = [
```

Comment:
yep - maybe create a `config.py` and define defaults in there. because currently defaults defined in argparser would override anything in `config.json` which doesn't make sense